File: models/entities/Candidate.js
import { CANDIDATE_STATUS } from "../../components/candidate/CandidateTabController.js";

export class Candidate{
    /**
     * 
     * @param {number} id 
     * @param {string} firstname 
     * @param {string} lastname 
     * @param {string} patronymic 
     * @param {string} email 
     * @param {string} phone 
     * @param {CANDIDATE_STATUS} status 
     */
    constructor(id, firstname, lastname, patronymic, email, phone, status){
        this.ID = id;
        this.firstname = firstname
        this.lastname = lastname
        this.patronymic = patronymic
        this.email = email
        this.phone = phone
        this.status = status
    }
}

File: revel-app/app/models/entities/Candidate.go
package entities

//Candidate структура сущности кандидата
type Candidate struct {
	ID         int64  `json:"ID"`
	Firstname  string `json:"firstname"`
	Lastname   string `json:"lastname"`
	Patronymic string `json:"patronymic"`
	Email      string `json:"email"`
	Phone      string `json:"phone"`
	Status     string `json:"status"`
}
